From fed910a304a301bdc854f61f14b586e3923d8322 Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Fri, 5 Sep 2025 12:41:42 +0200
Subject: [PATCH] patch

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 11af73e..cf2b50f 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -34,7 +34,7 @@ dependencies = [
     "ipython >=7, <10",
     "pydantic >=2 , <3",
     "pydantic-extra-types >=2, <3",
-    "enum-tools==0.12.0"
+    "enum-tools>=0.12.0"
 ]
 requires-python = ">=3.9"
 
-- 
2.39.5 (Apple Git-154)

