From 64237fe5dd2159232eada2fdbb96d712d648a5e0 Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Thu, 4 Sep 2025 12:00:44 +0200
Subject: [PATCH] path

---
 python-wrapper/pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python-wrapper/pyproject.toml b/python-wrapper/pyproject.toml
index 11af73e..cf2b50f 100644
--- a/python-wrapper/pyproject.toml
+++ b/python-wrapper/pyproject.toml
@@ -34,7 +34,7 @@ dependencies = [
     "ipython >=7, <10",
     "pydantic >=2 , <3",
     "pydantic-extra-types >=2, <3",
-    "enum-tools==0.12.0"
+    "enum-tools>=0.12.0"
 ]
 requires-python = ">=3.9"
 
-- 
2.39.5 (Apple Git-154)

